SQL/tgstation_schema_prefixed.sql

```diff
@@ -148,6 +148,7 @@ CREATE TABLE `SS13_death` (
   `toxloss` smallint(5) unsigned NOT NULL,
   `cloneloss` smallint(5) unsigned NOT NULL,
   `staminaloss` smallint(5) unsigned NOT NULL,
+  `last_words` varchar(96) DEFAULT NULL
```

Comment:
Theoretically, a deathgasp message can be much longer than this. It's hard capped to 102 characters, but then, random ellipses are added on top of that. Thoughts?